quadriga/src/main/java/edu/asu/diging/quadriga/core/conceptpower/model/ConceptCache.java

```diff
@@ -0,0 +1,223 @@
+        return conceptList;
+    }
+
+    public void setConceptList(String conceptList) {
+        this.conceptList = conceptList;
+    }
+
+    public String getTypeId() {
+        return typeId;
+    }
+
+    public void setTypeId(String typeId) {
+        this.typeId = typeId;
+    }
+
+    public boolean isDeleted() {
+        return deleted;
+    }
+
+    public void setDeleted(boolean deleted) {
+        this.deleted = deleted;
+    }
+
+    public LocalDateTime getLastUpdated() {
+        return lastUpdated;
+    }
+
+    public void setLastUpdated(LocalDateTime lastUpdated) {
+        this.lastUpdated = lastUpdated;
+    }
+
+    public List<String> getAlternativeUris() {
+        return alternativeUris;
+    }
+
+    public void setAlternativeUris(List<String> alternativeUris) {
+        this.alternativeUris = alternativeUris;
+    }
+
+    public List<String> getEqualTo() {
+        return equalTo;
+    }
+
+    public void setEqualTo(List<String> equalTo) {
+        this.equalTo = equalTo;
+    }
+
+    public List<String> getWordNetIds() {
+        return wordNetIds;
+    }
+
+    public void setWordNetIds(List<String> wordNetIds) {
+        this.wordNetIds = wordNetIds;
+    }
+
+    public String getCreatorId() {
+        return creatorId;
+    }
+
+    public void setCreatorId(String creatorId) {
+        this.creatorId = creatorId;
+    }
+
+    public ConceptType getConceptType() {
+        return conceptType;
+    }
+
+    public void setConceptType(ConceptType conceptType) {
+        this.conceptType = conceptType;
+    }
+
+    @Override
+    public int compareTo(ConceptCache conceptCache) {
+        // If both old and new cache values are null/blank/empty, nothing has changed
+        // If old value is null/blank/empty, new value is not null/blank/empty, difference present
+        // If old value is not null/blank/empty, new value is null/blank/empty, difference present
+        // If both are not null/blank/empty, we need to check difference
+        
+        if(isDifferentList(conceptCache.getAlternativeUris(), this.getAlternativeUris())) return -1;
+        if(isDifferentList(conceptCache.getEqualTo(), this.getEqualTo())) return -1;
+        if(isDifferentList(conceptCache.getWordNetIds(), this.getWordNetIds())) return -1;
+        if (isDifferentString(conceptCache.getConceptList(), this.getConceptList())) return -1;
+        if (isDifferentString(conceptCache.getCreatorId(), this.getCreatorId())) return -1;
+        if (isDifferentString(conceptCache.getDescription(), this.getDescription())) return -1;
+        if (isDifferentString(conceptCache.getId(), this.getId())) return -1;
+        if (isDifferentString(conceptCache.getPos(), this.getPos())) return -1;
+        if (isDifferentString(conceptCache.getTypeId(), this.getTypeId())) return -1;
+        if (isDifferentString(conceptCache.getUri(), this.getUri())) return -1;
+        if (isDifferentString(conceptCache.getWord(), this.getWord())) return -1;
```

Comment:
this is not following our formatting guidelines. or it could be put as an OR statement (formatted one condition a line), so that only once -1 is returned.